app/presenters/outcome_presenter.rb

```diff
@@ -31,40 +55,60 @@ def has_calendar?
     calendar.present?
   end
 
-  def has_body?
-    use_template? || super()
-  end
-
   def body
-    if use_template?
-      view_context = @state.dup
```

Comment:
Hmm, good question. I think it was to avoid us changing the state object but I don't remember for sure. Let me dig into this to work out why we added it!